.markdownlint.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This ... isn't a file we read.

Perhaps you're confusing this repo with the javascript version of markdownlint?